lang/pl_PL.json

```diff
@@ -0,0 +1,347 @@
+    "<0>Send a message to DevCompliance</0> for:": "<0>Wyślij wiadomość do DevCompliance</0> aby:",
+    "<0>Send a message to QATeam</0> for:": "<0>Wyślij wiadomość do QATeam</0> aby:",
+    "<0>Send a message to RAArtTeam</0> for:": "<0>Wyślij wiadomość do RAArtTeam</0> aby:",
+    "<0>Send a message to DevQuest</0> for submissions, questions, ideas, or reporting issues related to <1>DevQuest</1>.": "<0>Wyślij wiadomość do DevQuest</0> w celu przesłania raportów, pytań, pomysłów lub zgłoszeń o problemach związanych z <1>DevQuest</1>.",
+    "<0>Send a message to RACheats</0> if you believe someone is in violation of our <1>Global Leaderboard and Achievement Hunting Rules</1>.": "<0>Wyślij wiadomość do RACheats</0> jeśli uważasz, że ktoś naruszył nasze <1>Zasady globalnych tabel wyników i Zasady polowań na osiągnięcia</1>.",
+    "<0>Send a message to WritingTeam</0> for:": "<0>Wyślij wiadomość do WritingTeam</0> aby:",
+    "<0>Send a message to RANews</0> for:": "<0>Wyślij wiadomość do RANews</0> aby:",
+    "<0>Send a message to RAEvents</0> for submissions, questions, ideas, or reporting issues related to <1>community events</1>.": "<0>Wyślij wiadomość do RAEvents</0> w celu przesłania raportów, pytań, pomysłów lub zgłoszeń o problemach związanych z <1>wydarzeniami społeczności</1>.",
+    "<0>Send a message to TheUnwanted</0> for submissions, questions, ideas, or reporting issues specifically related to <1>The Unwanted</1>.": "<0>Send a message to TheUnwanted</0> w celu przesłania raportów, pytań, pomysłów lub zgłoszeń o problemach związanych konkretnie z <1>The Unwanted</1>.",
+    "By signing up on RetroAchievements.org, you agree to the <0>Users Code of Conduct</0>.": "Poprzez rejestrację na RetroAchievements.org zgadzasz się z <0>Kodeksem Postępowania Użytkowników</0>.",
+    "By joining the Junior Developer and/or Developer team, you agree to the <0>Developers Code of Conduct</0>.": "Poprzez dołączenie do zespołu Junior Deweloperów i/lub Deweloperów zgadzasz się z <0>Kodeksem Postępowania Deweloperów</0>.",
+    "Achievements": "Osiągnięcia",
+    "Last Updated": "Ostatnio zaktualizowane",
+    "Tickets": "Zgłoszenia",
+    "Leaderboards": "Tabele wyników",
+    "Progress": "Postęp",
+    "Players": "Gracze",
+    "Points": "Punkty",
+    "unknown": "nieznany",
+    "Release Date": "Data wydania",
+    "Rarity": "Rzadkość",
+    "Title": "Tytuł",
+    "Select a game": "Wybierz grę",
+    "Add to Want to Play Games": "Dodaj do listy Chcę w to zagrać",
+    "Remove from Want to Play Games": "Usuń z listy Chcę w to zagrać",
+    "Ascending (A - Z)": "Rosnąco (A - Z)",
+    "Descending (Z - A)": "Malejąco (Z - A)",
+    "Earliest": "Najwcześniejsze",
+    "Latest": "Najpóźniejsze",
+    "Less": "Mniej",
+    "Both": "Oba",
+    "Customize View": "Dopasuj widok",
+    "Playable": "Grywalne",
+    "Not Playable": "Niegrywalne",
+    ":count System|:count Systems": ":count system|:count systemy",
```

Comment:
Definitely outside of my knowledge, but just curious: should `system` and/or `systemy` be capitalized?

Reply:
That's one of the things I was contemplating about. In Polish, we capitalize only the first word of the sentence, or if the word is a "proper name" (firstname, surname, brand, city name etc.). It's different than for example German, where every noun is capitalized.
In Polish journalism, we also don't use popular on the west "heading rules", in which most of the words starts with capitalized letter.

By the way, I was also thinking if I should use word "System" or "Platform", as the latter one is in Poland more often used in regards to consoles (like PS or Xbox). I think that the meaning of these two is similar as in English. Though, if RA uses word "System" for naming it, I followed this manner.

Reply:
That is very interesting. Thank you for the explanation!

> By the way, I was also thinking if I should use word "System" or "Platform", as the latter one is in Poland more often used in regards to consoles (like PS or Xbox). I think that the meaning of these two is similar as in English.

If the Polish word for "Platform" has more cultural significance, that definitely seems like it would be more appropriate to use. pt_BR had a similar situation: they chose to use "Platinado" as the word for mastery as it had more cultural significance than the direct translation of "Mastered".

Happy to hold off on merging if you'd like to make this change in this branch.

Reply:
I'm not sure if I should make that change. I think that "System" could still fit in there, and that's what RA uses originally (and in URL schema as well). Optionally, we could left it as is, and if Polish community would prefer changing it later to "Platform", the change could be done.

Reply:
It is totally your choice! 🙂
For now, shall I take the branch as-is?

Reply:
Yeah, I think that at this state it is ready to take off.